hrm-domain/scheduled-tasks/hrm-data-pull/pull-cases.ts

```diff
@@ -51,15 +51,7 @@ export const pullCases = async (startDate: Date, endDate: Date) => {
     };
   });
 
-  const mapContactsToId = (contacts: Required<{ id: number }>[]) =>
```

Comment:
@GPaoloni I think we need to keep the case -> contacts references for now, until we've confirmed with KHP that they're not relying on them already.

Reply:
Thanks for pointing out, just reverted that change.